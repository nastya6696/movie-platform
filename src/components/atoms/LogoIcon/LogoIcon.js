import PropTypes from 'prop-types';

import styles from './styles.module.scss';

export const LogoIcon = ({boldPart, thinPart}) => (
  <div className={styles.logoText}>
    <strong className={styles.boldText}>{boldPart}</strong>{thinPart}
  </div>
)

LogoIcon.propTypes = {
  boldPart: PropTypes.string.isRequired,
  thinPart: PropTypes.string.isRequired
}

