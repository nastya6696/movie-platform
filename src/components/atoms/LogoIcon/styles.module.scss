@import "src/assets/scss/common";

.logoText {
  @include fontSizes(20px, 25px);
  color: $red;
  font-weight: 300;
}

.boldText {
  font-weight: 900;
}
