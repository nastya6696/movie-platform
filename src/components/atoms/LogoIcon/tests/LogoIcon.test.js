import { create } from 'react-test-renderer';
import {LogoIcon} from "../LogoIcon";

describe('LogoIcon', () => {
  it('should create simple LogoIcon component', () => {
    const component = create(<LogoIcon thinPart='thinPart' boldPart='boldPart' />);
    expect(component).toMatchSnapshot();
  })
})
