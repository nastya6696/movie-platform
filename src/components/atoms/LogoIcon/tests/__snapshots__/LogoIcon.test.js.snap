// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogoIcon should create simple LogoIcon component 1`] = `
<div
  className="logoText"
>
  <strong
    className="boldText"
  >
    boldPart
  </strong>
  thinPart
</div>
`;
